packages/apps/src/Apps.tsx

```diff
@@ -26,7 +26,7 @@ interface SidebarState {
 
 function WarmUp (): React.ReactElement {
   const { api, isApiReady } = useApi();
-  const fees = useCall<any>(isApiReady ? api.derive.balances?.fees : undefined, []);
+  const fees = null;
```

Comment:
Sorry about being picky, but I am thinking about how we could easily identify and track changes differing from `polkadot/apps`. 

Instead of deleting the code generated by polkadot, how about we comment them and add our code in new lines?

Reply:
sure.. will comment the code

Reply:
@KarishmaBothara I searched around the codebase, and found many places are using `fees.`, but did not search deeply and not sure if `fees` here was passed down to components, and used as `fees.***`?